pyspecProcScripts/load_data.py

```diff
@@ -656,31 +637,6 @@ def proc_spincore_ODNP_v3(s, fl=None):
     return s
 
 
-def proc_spincore_ODNP_v4(s, fl=None):
-    if "t" in s.dimlabels:
-        s.chunk("t", ["ph2", "ph1", "t2"], [4, 4, -1])
-        s.set_units("t2", "s")
-    s.rename("power", "time")
-    s.setaxis("ph1", r_[0, 1, 2, 3.0] / 4)
-    s.setaxis("ph2", r_[0, 1, 2, 3.0] / 4)
-    s.set_prop("coherence_pathway", {"ph1": 1, "ph2": -2})
-    s.set_units("t2", "s")
-    s["t2"] -= s.get_prop("acq_params")["tau_us"] * 1e-6
-    s *= s.shape["nScans"]
-    s.squeeze()
-    s.ft("t2", shift=True)
-    s.ft(["ph1", "ph2"])
-    s.reorder(["ph1", "ph2", "time"])
-    if fl is not None:
-        fl.next("Raw Data \n Frequency Domain")
-        fl.image(s)
-        s.ift("t2")
-        fl.next("Raw Data \n Time Domain")
-        fl.image(s)
-        s.ft("t2")
-    return s
-
-
```

Comment:
this actually didn't run b/c the data doesn't have a `power` dimension, so it dies on line 663.  But also, I don't see any added value here on top of the generalproc function -- at all.

This does beg the question of whether we want to handle the business of turning the log into a power axis as part of the postproc, here, or if we want to handle it independently.  That could be a point for discussion

Reply:
but for now, it's clear this function offers no value any longer, and should die.

Reply:
not sure why your examples would be calling this version of postproc since the most recent version in examples/combined_ODNP calls v3 of IR and v5 of ODNP